freeipa/src/main/java/com/sequenceiq/freeipa/service/config/RootCertRegisterService.java

```diff
@@ -78,16 +98,7 @@ private RootCert createRootCertEntity(Stack stack, String rootCertificate) {
         rootCert.setStack(stack);
         rootCert.setEnvironmentCrn(stack.getEnvironmentCrn());
         rootCert.setCert(convertToPemFormat(rootCertificate));
-        return rootCert;
-    }
-
-    @Override
-    public void delete(Stack stack) {
-        try {
-            rootCertService.deleteByStack(stack);
-        } catch (NotFoundException e) {
-            LOGGER.info("Root cert not exists for environment {}", stack.getEnvironmentCrn());
-        }
+        return rootCertService.save(rootCert);
```

Comment:
please add a log line to this method so we would know if this was called from line 79